Python/ceval.c

```diff
@@ -1745,6 +1742,13 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
                 first_instr = frame->f_code->co_firstinstr;
                 next_instr = first_instr + nexti;
             }
+            JUMP_TO_INSTRUCTION(RESUME_QUICK);
+        }
+
+        TARGET(RESUME_QUICK) {
```

Comment:
Could you name for what it does, rather than how we got to it?
Maybe `CHECK_EVAL_BREAKER`?